specifyweb/frontend/js_src/lib/localization/resources.ts

```diff
@@ -450,4 +447,13 @@ export const resourcesText = createDictionary({
   saveFormFirstDescription: {
     'en-us': 'You need to save this form before you edit another one',
   },
+  defaultTableFormatter: {
+    'en-us': 'Default Table Formatter',
+  },
+  conditionalFormatter: {
+    'en-us': 'Conditional Formatter',
+  },
+  showFieldFormatters: {
+    'en-us': 'Show Field Formatters',
```

Comment:
While I like the fact that the non-default formatter option is now not always displayed to make UI simpler for common cases, I feel like we could better describe the role of this checkbox than "Show Field Formatters"

maybe "Customize Field Formatters" or "Use non-default field formatters" or "Use Custom Field Formatters" (because that's effectively what it's doing - the default value Formatter will be used unless you customize it)

Reply:
ALSO, I don't remember this too fwell, but doesn't this UI also allow you to pick which table formatter you want to use, rather than just what field formatter you want to use?
i.e if in the "Field" column you select a -to-one relationship?

then also if you select a -to-many relationship, does the UI offer selecting which aggregator to use?